website/content/docs/oss/configuration/worker/kms-worker.mdx

```diff
@@ -0,0 +1,105 @@
+---
+layout: docs
+page_title: KMS Worker Configuration
+description: |-
+  KMS worker-specific parameters.
+---
+
+
+## KMS Worker Configuration
+
+KMS Workers require a `name` field. This specifies a unique name of this worker within the Boundary cluster.
+The `name` value can be:
+- a direct name string (must be all lowercase)
+- a reference to a file on disk (`file://`) from which the name is read
+- an env var (`env://`) from which the name is read.
+
```

Comment:
Might be good to call out that changing the name will result in the worker being considered a "new" worker in the cluster and being assigned a different worker ID.

Reply:
I added a note to address this